inspire/config.py

```diff
@@ -313,6 +314,11 @@
 
 SEARCH_ELASTIC_AGGREGATIONS = {
     "hep": {
+        "subject": {
+            "terms": {
+                "field": "subject_terms.scheme"
```

Comment:
I don't think this will be correct. In the `subject_terms.scheme` field we store where the subject term comes from (like `INSPIRE`) but what we want to show in the values is the actual values (`subject_terms.term`)

The tricky part is that we don't want to show all of the `subject_terms.term` (only the ones that have INSPIRE as a `scheme`) so here we will need to add a new field to the json document only with the values we want. I will look for an example on how to extend the json before indexing and you can use it.